src/calculateRentalCost.js

```diff
@@ -28,7 +28,13 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  if (days < 3) {
+    return days * 40;
+  } else if (days > 6) {
```

Comment:
```suggestion
  } else if (days >= 7) {
```
It's more meaningful